spec/workflow/client/workflow_template_spec.rb

```diff
@@ -10,11 +10,22 @@
 
   describe '#retrieve' do
     subject(:workflow_template) { routes.retrieve('accessionWF') }
+    let(:data) { '{"processes":[{"name":"start-assembly"},{"name":"content-metadata-create"}]}' }
```

Comment:
is this necessary, since it seems to have the same value as the `let` being overridden from the surrounding context?

Reply:
Nice catch. I intended to move it here, but I had forgotten to delete the original.  Fixed now.